resources/custom-resources/keycloak-setup/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/KeycloakSetup.java

```diff
@@ -156,11 +162,46 @@ protected boolean importRealm(String keycloakHost, String bearerToken, String re
         ));
         adminUser.put("requiredActions", List.of("UPDATE_PASSWORD"));
 
+        // Public OAuth app client with PKCE for the admin web app
+        String adminWebAppClientId = Utils.randomString(20);
+        String adminWebAppClientName = realmName + "-admin-webapp-client";
+        LinkedHashMap<String, Object> adminWebAppClient = new LinkedHashMap<>();
+        adminWebAppClient.put("enabled", true);
+        adminWebAppClient.put("protocol", "openid-connect");
+        adminWebAppClient.put("name", adminWebAppClientName);
+        adminWebAppClient.put("clientId", adminWebAppClientId);
+        adminWebAppClient.put("description", "SaaS Boost Admin Web App Client");
+        adminWebAppClient.put("redirectUris", List.of(adminWebAppUrl));
```

Comment:
non-blocking: would be nice to also include `localhost:3000` for testing

Reply:
redirectUris should also include a wildcard, e.g. `https://randomstring.cloudfront.net*` to cover redirects towards ui sub-pages, like `https://randomstring.cloudfront.net/dashboard#`

Reply:
we also need to configure CORS for access through this client: configuring the [`webOrigins` key](https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_clientrepresentation) for `*` will solve this